hack/vendor.sh

```diff
@@ -1,28 +1,28 @@
 #!/usr/bin/env bash
 
-# This file is just wrapper around vndr (github.com/LK4D4/vndr) tool.
-# For updating dependencies you should change `vendor.conf` file in root of the
-# project. Please refer to https://github.com/LK4D4/vndr/blob/master/README.md for
-# vndr usage.
+# This file is just wrapper around 'go mod vendor' tool.
+# For updating dependencies you should change `vendor.mod` file in root of the
+# project.
 
 set -e
+set -x
 
-if ! hash vndr; then
-	echo "Please install vndr with \"go get github.com/LK4D4/vndr\" and put it in your \$GOPATH"
-	exit 1
+SCRIPTDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+${SCRIPTDIR}/go-mod-prepare.sh
+
+if [ $# -eq 0 ] || [ "$1" != "archive/tar" ]; then
+	GO111MODULE=auto go mod vendor -modfile vendor.mod
 fi
 
 if [ $# -eq 0 ] || [ "$1" = "archive/tar" ]; then
-	echo "update vendored copy of archive/tar"
-	: "${GO_VERSION:=$(awk -F '[ =]' '$1 == "ARG" && $2 == "GO_VERSION" { print $3; exit }' ./Dockerfile)}"
-	rm -rf vendor/archive
-	mkdir -p ./vendor/archive/tar
-	echo "downloading: https://golang.org/dl/go${GO_VERSION%.0}.src.tar.gz"
-	curl -fsSL "https://golang.org/dl/go${GO_VERSION%.0}.src.tar.gz" \
-		| tar --extract --gzip --directory=vendor/archive/tar --strip-components=4 go/src/archive/tar
-	patch --strip=4 --directory=vendor/archive/tar --input="$PWD/patches/0001-archive-tar-do-not-populate-user-group-names.patch"
+        echo "update vendored copy of archive/tar"
+        : "${GO_VERSION:=$(awk -F '[ =]' '$1 == "ARG" && $2 == "GO_VERSION" { print $3; exit }' ./Dockerfile)}"
+        rm -rf vendor/archive/tar
+        mkdir -p vendor/archive/tar
+        echo "downloading: https://golang.org/dl/go${GO_VERSION%.0}.src.tar.gz"
+        curl -fsSL "https://golang.org/dl/go${GO_VERSION%.0}.src.tar.gz" \
+                | tar --extract --gzip --directory=vendor/archive/tar --strip-components=4 go/src/archive/tar
+        patch --strip=4 --directory=vendor/archive/tar --input="$PWD/patches/0001-archive-tar-do-not-populate-user-group-names.patch"
+        rm -f vendor/archive/tar/*_test.go
```

Comment:
Small nit and maybe not necessary but we should use the `find` cmd in case of subpkg in the future:

```console
find ./vendor/archive/tar -name "*_test.go" -type f -exec rm -f {} \;
```